scripts/qa_crowbarsetup.sh

```diff
@@ -633,6 +633,14 @@ function onadmin_prepare_cloud_repos()
                 ;;
         esac
     fi
+
+    # Ensure Pool/Updates repos are always configured on admin server
+    for repo in Pool Updates; do
+        name="SUSE-Cloud-$(echo $cloudsource | sed "s/[^0-9]//g")-$repo"
```

Comment:
Is this name relevant or does it only need to be unique?
Otherwise it could cause trouble with GM, GMC or M4